config/fcos/v1_5_exp/translate.go

```diff
@@ -57,33 +57,33 @@ const (
 	bootV1SizeMiB = 384
 )
 
-// ToIgn3_3Unvalidated translates the config to an Ignition config.  It also
+// ToIgn3_4Unvalidated translates the config to an Ignition config.  It also
 // returns the set of translations it did so paths in the resultant config
 // can be tracked back to their source in the source config.  No config
 // validation is performed on input or output.
-func (c Config) ToIgn3_3Unvalidated(options common.TranslateOptions) (types.Config, translate.TranslationSet, report.Report) {
-	ret, ts, r := c.Config.ToIgn3_3Unvalidated(options)
+func (c Config) ToIgn3_4Unvalidated(options common.TranslateOptions) (types.Config, translate.TranslationSet, report.Report) {
+	ret, ts, r := c.Config.ToIgn3_4Unvalidated(options)
 	if r.IsFatal() {
 		return types.Config{}, translate.TranslationSet{}, r
 	}
 	r.Merge(c.processBootDevice(&ret, &ts, options))
 	return ret, ts, r
 }
 
-// ToIgn3_3 translates the config to an Ignition config.  It returns a
+// ToIgn3_4 translates the config to an Ignition config.  It returns a
 // report of any errors or warnings in the source and resultant config.  If
 // the report has fatal errors or it encounters other problems translating,
 // an error is returned.
-func (c Config) ToIgn3_3(options common.TranslateOptions) (types.Config, report.Report, error) {
-	cfg, r, err := cutil.Translate(c, "ToIgn3_3Unvalidated", options)
+func (c Config) ToIgn3_4(options common.TranslateOptions) (types.Config, report.Report, error) {
+	cfg, r, err := cutil.Translate(c, "ToIgn3_4Unvalidated", options)
 	return cfg.(types.Config), r, err
 }
 
-// ToIgn3_3Bytes translates from a v1.4 Butane config to a v3.3.0 Ignition config. It returns a report of any errors or
+// ToIgn3_4Bytes translates from a v1.5 Butane config to a v3.4.0 Ignition config. It returns a report of any errors or
```

Comment:
OOC: Is the move to targeting 3.4.0-experimental a given when creating a new experimental Butane config? I'd think it would be a more "lazy" approach where we keep targeting the same version until we have to move in case e.g. we add new Butane sugar which doesn't need new Ignition.

Reply:
It's not a given.  I've generally assumed that we want to be developing against the experimental Ignition spec, and can always roll back to the stable Ignition one before stabilizing the Butane spec.  But you're right, it could make sense to go the other way.